src/module/file/File.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * kintone api - js client
+ */
+import {Connection} from '../connection/Connection';
+/**
+ * File module
+ */
+import {File as FileModule} from 'kintone-basejs-sdk';
+
+export class File extends FileModule{

```

Comment:
Need to fix eslint in this line

Reply:
fixed in commit below:
https://github.com/kintone/kintone-js-sdk/commit/8f1a0e80f55611890462c6a2ce620d9a2cce72db